src/main/js/store/actions/globalConfiguration.js

```diff
@@ -153,18 +157,39 @@ function clearFieldErrors() {
     };
 }
 
-function handleFailureResponse(dispatch, responseData, statusCode) {
+function validatingConfig() {
+    return {
+        type: CONFIG_VALIDATING
+    };
+}
+
+function validatedConfig() {
+    return {
+        type: CONFIG_VALIDATED
+    };
+}
+
+function configValidationError({ message, errors }) {
+    return {
+        type: CONFIG_VALIDATE_ERROR,
+        message,
+        errors
+    };
+}
+
+function createErrorHandler(type, defaultHandler) {
     const errorHandlers = [];
-    const configErrorHandler = () => configError(responseData.message, responseData.errors);
     errorHandlers.push(HTTPErrorUtils.createUnauthorizedHandler(unauthorized));
-    errorHandlers.push(HTTPErrorUtils.createDefaultHandler(() => configError(responseData.message, null)));
-    errorHandlers.push(HTTPErrorUtils.createBadRequestHandler(configErrorHandler));
-    errorHandlers.push(HTTPErrorUtils.createPreconditionFailedHandler(configErrorHandler));
-    const handler = HTTPErrorUtils.createHttpErrorHandler(errorHandlers);
-    dispatch(handler(statusCode));
+    errorHandlers.push(HTTPErrorUtils.createForbiddenHandler(() => configError({ type, message: HTTPErrorUtils.MESSAGES.FORBIDDEN_ACTION })));
+    errorHandlers.push(HTTPErrorUtils.createBadRequestHandler(defaultHandler));
+    errorHandlers.push(HTTPErrorUtils.createPreconditionFailedHandler(defaultHandler));
+    errorHandlers.push(HTTPErrorUtils.createDefaultHandler(defaultHandler));
+
+    return HTTPErrorUtils.createHttpErrorHandler(errorHandlers);
 }
 
-export function refreshConfig(id) {
+export function
+refreshConfig(id) {
```

Comment:
Do we prefer a multiline method signature?

Reply:
Why is `export function` and `refreshConfig(id) {` on different lines?

Reply:
This was a mistake, I can update it